Assistant UI updates

rosannamilner opened the pull request:
General
- Removed extra unnecessary cards and dividers
- added missing warning statuses

Updated form
- Changed URL form to use submit button only
- Changed display for File Selected

Improved footer and moved to new file
- added translate, copy and expand/minimise buttons 
- added detected language in full

Credibility Signals
- added new footers
- corrected previous fact check behaviour when none returned

Extracted URLs
- open in new tab not same window
- aligned loading symbol with warning/mention/positive credibility icons

Media card
- using full width

Sallaa commented:
Thank you for the changes, the interface looks much better / cleaner already. 

It looks like there is a display issue with the submit button in the form when the window is resized to a smaller width

<img width="1171" alt="Screenshot 2024-09-23 at 14 13 12" src="https://github.com/user-attachments/assets/947fae5d-4649-4693-868d-d001e2c8f127">

rosannamilner commented:
> Thank you for the changes, the interface looks much better / cleaner already.
> 
> It looks like there is a display issue with the submit button in the form when the window is resized to a smaller width
> 
> <img alt="Screenshot 2024-09-23 at 14 13 12" width="1171" src="https://private-user-images.githubusercontent.com/28429921/369867455-947fae5d-4649-4693-868d-d001e2c8f127.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc2ODU0MTksIm5iZiI6MTcyNzY4NTExOSwicGF0aCI6Ii8yODQyOTkyMS8zNjk4Njc0NTUtOTQ3ZmFlNWQtNDY0OS00NjkzLTg2OGQtZDAwMWUyYzhmMTI3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTMwVDA4MzE1OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI4NmEyZmVkNGVlNmZkMmEwNGY4YjVmNWEyMzczODhlYmI2MzJlOTM2ODZjMDI2YTE1MjAxNjg4NDE0Mjg0MzImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Rbsc5HK4o-vh3MNrX_X80k4r3zc_jphL4SD3qZXp_pY">

Thanks for spotting! I have changed to use Stack instead of Grid2 which has solved this problem. I have also fixed two bugs I spotted in AssistantCredibilitySignals.jsx related to the new text footer. This is ready to be merged now if you're happy.

twinkarma reviewed (approved):
LGTM, the UI definitely looks a lot cleaner!
